src/Drivers/hiopbbpy/BODriverEX.py

```diff
@@ -57,9 +57,9 @@ def con_jac_ineq(x):
 for prob_type in prob_type_l:
    print()
    if prob_type == "LpNorm":
-      problem = LpNormProblem(nx, xlimits, constraints=None)
+      problem = LpNormProblem(nx, xlimits)
    else:
-      problem = BraninProblem(constraints=None)
+      problem = BraninProblem()
    problem.set_constraints(user_constraint)
```

Comment:
I have two ways to add the constraints:
1. in the constructor, setting `constraints=user_constraint`
or
2. using function `set_constraints`.
I should leave a comment in this example.